### DISCUSSION TITLE
Sidecar mode and Ambient Mesh

### DISCUSSION THREAD

**Discussion by Romero027** [istio/istio#48395] [Q&A] [answered]
Can sidecars and ambient mesh coexist in the same cluster/namespace? For example, can I configure some of the services to use a waypoint proxy while other services still use the sidecars? If so, is there a sample configuration?

Thanks in advance!

**Comment by howardjohn** [comment, marked as answer]
More or less yes. There is some existing support, which should "just work" if you try it. However, extensive testing, documentation, etc has been deferred until post-beta, and we are focusing on stabilizing the pure-ambient use case first.

**Reply by Romero027** [reply]
Got it, thanks!
